logging-modules/flogger/src/test/java/com/baeldung/flogger/FloggerIntegrationTest.java

```diff
@@ -3,6 +3,7 @@
 import com.google.common.flogger.FluentLogger;
```

Comment:
let's remove unused tests

Reply:
If it's because of the import _import org.junit.Ignore_, I have removed this import. Currently, there are no unused tests.